### ISSUE TITLE
enabled configuration option is ignored

### ISSUE THREAD

**Issue by modosc** [rollbar/rollbar.js#280]
there are only a few references to it in the 2.x code:
```
laptop ~/git/rollbar.js/src [master|✔]$ grep -ri enabled *
browser/plugins/jquery.js:      if (rb.options && !rb.options.enabled) {
browser/plugins/jquery.js:        msg = '[Rollbar not enabled]';
browser/transforms.js:        cookie_enabled: window.navigator.cookieEnabled,
server/rollbar.js:  enabled: true
```
and as far as i can tell it's not doing anything:
![image](https://cloud.githubusercontent.com/assets/2231664/26333373/f5711fea-3f11-11e7-928f-5afe67c2b023.png)

**Comment by rokob** [comment]
Yikes

**Comment by modosc** [comment]
can you roll a release to include this please?

**Comment by rokob** [comment]
Released 2.0.2 with this fix, it might take a bit to propagate to CDNJS

**Comment by 94marianag** [comment]
I'm still having this issue with version 2.26.2, I set the variable enabled to false, and still sending data to rollbar

`const Rollbar = require('rollbar');

const rollbar = new Rollbar({
    accessToken: process.env.ROLLBAR_ACCESS_TOKEN,
    captureUncaught: true,
    captureUnhandledRejections: true,
    payload: {
        code_version: '1.0.0',
    }
});

rollbar.configure({
    enabled: process.env.ROLLBAR_SERVICE_ENABLED,
});
module.exports=rollbar;
`
